scripts/vet.sh

```diff
@@ -184,4 +184,11 @@ revive -formatter plain ./... >"${REV_OUT}" || true
 # TODO: Remove `|| true` to unskip linter failures once existing issues are fixed.
 (noret_grep -v "unused-parameter" "${REV_OUT}" | not grep -v "\.pb\.go:") || true
 
+# Collection of trailing spaces analysis checks
+TRAIL_SPACE_OUT="$(mktemp)"
+find . -type f -exec grep -Hn "[[:blank:]]$" {} \; >"${TRAIL_SPACE_OUT}"
+
+# Error for anything other than in .git directory, vendor directory and *.md files.
+noret_grep -v "./.git" "${TRAIL_SPACE_OUT}" | noret_grep -v "vendor" | not grep -v ".md"
```

Comment:
Can we avoid scanning these files instead of excluding them later? `find` has a `-prune` flag that may be useful.

Reply:
Done